src/components/explorer/Acc.js

```diff
@@ -73,6 +73,9 @@ const Acc = ({ match }) => {
                 case 'dataAccount':
                     props.data.adi = ParseADI(props.data.data.url);
                     return <DataAccount data={props.data} />;
+                case 'liteDataAccount':
```

Comment:
Are there any differences with `dataAccount`? Could not we use the same module `DataAccount` for `liteDataAccounts`?